packages/coinstac-client-core/package.json

```diff
@@ -41,7 +41,9 @@
     "commander": "^2.9.0",
     "convict": "^1.5.0",
     "csv-parse": "^1.2.0",
+    "cycle": "^1.0.3",
```

Comment:
It was reported by the linter for not being included in the `package.json`, but being imported in one of the files (`dynamic-api.js`).